tests/monitoring_test.go

```diff
@@ -406,6 +657,26 @@ func waitForPrometheusAlert(f *framework.Framework, alertName string) {
 	}, 10*time.Minute, 1*time.Second).Should(BeTrue())
 }
 
+func waitForNoPrometheusAlert(f *framework.Framework, alertName string) {
```

Comment:
very similar to waitForPrometheusAlert maybe can extract some part and negate the outcome?

not must for this PR of course

Reply:
Right, will do.